### PR TITLE
✨ TWEAK: improve delay expression

### PR DESCRIPTION
**Pull request by warengonzaga**
Closing #1507 replace the delay expression from `s` to `x`
Update also the documentation regarding this change.

### CODE REVIEW

File: animate.css [hunk 1]
```diff
@@ -42,31 +42,31 @@
   -webkit-animation-iteration-count: calc(var(--animate-repeat) * 3);
   animation-iteration-count: calc(var(--animate-repeat) * 3);
 }
-.animate__animated.animate__delay-1s {
+.animate__animated.animate__delay-1x {
```
Comment (eltonmesquita) [inline animate.css:45]: Could we support the old and the new classes?
```suggestion
.animate__animated.animate__delay-1x,
.animate__animated.animate__delay-1s {

File: animate.css [hunk 2]
```diff
@@ -42,31 +42,31 @@
...
   -webkit-animation-delay: 1s;
   animation-delay: 1s;
   -webkit-animation-delay: var(--animate-delay);
   animation-delay: var(--animate-delay);
 }
-.animate__animated.animate__delay-2s {
+.animate__animated.animate__delay-2x {
```
Comment (eltonmesquita) [inline animate.css:51]: ♻️  Same here

File: animate.css [hunk 3]
```diff
@@ -42,31 +42,31 @@
...
   -webkit-animation-delay: calc(1s * 2);
   animation-delay: calc(1s * 2);
   -webkit-animation-delay: calc(var(--animate-delay) * 2);
   animation-delay: calc(var(--animate-delay) * 2);
 }
-.animate__animated.animate__delay-3s {
+.animate__animated.animate__delay-3x {
```
Comment (eltonmesquita) [inline animate.css:57]: ♻️

File: animate.css [hunk 4]
```diff
@@ -42,31 +42,31 @@
...
   -webkit-animation-delay: calc(1s * 3);
   animation-delay: calc(1s * 3);
   -webkit-animation-delay: calc(var(--animate-delay) * 3);
   animation-delay: calc(var(--animate-delay) * 3);
 }
-.animate__animated.animate__delay-4s {
+.animate__animated.animate__delay-4x {
```
Comment (eltonmesquita) [inline animate.css:63]: ♻️

File: animate.css [hunk 5]
```diff
@@ -42,31 +42,31 @@
...
   -webkit-animation-delay: calc(1s * 4);
   animation-delay: calc(1s * 4);
   -webkit-animation-delay: calc(var(--animate-delay) * 4);
   animation-delay: calc(var(--animate-delay) * 4);
 }
-.animate__animated.animate__delay-5s {
+.animate__animated.animate__delay-5x {
```
Comment (eltonmesquita) [inline animate.css:69]: ♻️

File: docsSource/sections/02-utilities.md [hunk 6]
```diff
@@ -7,17 +7,17 @@ Animate.css comes packed with a few utility classes to simplify its use.
...
 | Class name          | Default delay time |
 | ------------------- | ------------------ |
-| `animate__delay-2s` | `2s`               |
-| `animate__delay-3s` | `3s`               |
-| `animate__delay-4s` | `4s`               |
-| `animate__delay-5s` | `5s`               |
+| `animate__delay-2x` | `2s`               |
+| `animate__delay-3x` | `3s`               |
+| `animate__delay-4x` | `4s`               |
+| `animate__delay-5x` | `5s`               |
```
Comment (eltonmesquita) [inline docsSource/sections/02-utilities.md:20]: How about adding a notice right under this table about both sorts of classes?
```suggestion
| `animate__delay-5x` | `5s` |

> In the previous versions, the library supported a slightly different version of the classes, like `animate__delay-2s`. As this utility works as a multiplier, it made it very confusing to use if the root variable `--animate-delay` was set to anything but `1`. We changed it but we still support the old classes and there's no need to migrate your code to the newer version.

**Review by eltonmesquita — CHANGES_REQUESTED** [review]
Just some comments to add support for the old way of doing it. Nice work mate 🙌!

File: docsSource/sections/02-utilities.md (hunk 6 above)
Comment (warengonzaga) [reply docsSource/sections/02-utilities.md:20]: Gotcha, @eltonmesquita!

**Comment by warengonzaga** [comment]
Hey @eltonmesquita, added support to old versions. Updated the docs too.

**Review by eltonmesquita — APPROVED** [review]
Nice one! Thanks for the changes. I'll get this merged into the `v5` branch so we can get it live in the next major version 🎉

**Comment by warengonzaga** [comment]
> Nice one! Thanks for the changes. I'll get this merged into the `v5` branch so we can get it live in the next major version 🎉

Excited to launch the v5! I'll work on other improvements ;)

**Pull request merged**